### ISSUE TITLE
[Discussion] FFMPEG Handling

### ISSUE THREAD

**Issue by mattdesl** [mattdesl/canvas-sketch#59]
I've added streaming MP4 and GIF exports with the `--stream` flag, which is pretty handy.

I'm curious about how I could integrate [@ffmpeg-intsaller/ffmpeg](https://www.npmjs.com/package/@ffmpeg-installer/ffmpeg) into this project without forcing it upon the user. Basically, giving users a single npm approach to installing ffmpeg, rather than telling them to install through brew or whatever, _however_ still allowing canvas-sketch to support the PATH-installed ffmpeg if needed. By default, it seems too heavy to add as a dependency to `canvas-sketch-cli`, which might get used in a per-project basis.

Basically looking for a UX solve... open to discussion!

**Comment by Apidcloud** [comment]
Not sure if what's you are looking for, but maybe having the binaries within the package could be a solution. Other known packages such as Canvas do so--though I think ffmpeg still needs to be installed. 

And if adding it as a dependency is too much, maybe have it as an external module that you download and install when needed for the 1st time--a quick but dirty way would be to inject that dependency into the package.json.
